packages/website/rulesMeta.ts

```diff
@@ -0,0 +1,15 @@
+import * as eslintPlugin from '@typescript-eslint/eslint-plugin';
+
+export const rulesMeta = Object.entries(eslintPlugin.rules).map(
+  ([name, content]) => ({
+    name,
+    type: content.meta.type,
+    docs: content.meta.docs,
+    fixable: content.meta.fixable,
+    hasSuggestions: content.meta.hasSuggestions,
```

Comment:
I have a question: is there any difference between `meta.hasSuggestions` and `meta.docs.suggestion`? The latter isn't mentioned in ESLint docs but we have it nonetheless.

Reply:
Hmmm, we don't use `meta.docs.suggestion` anymore, it seems